library/pk_wrap.c

```diff
@@ -994,11 +996,24 @@ static int pk_ecdsa_sig_asn1_from_psa( unsigned char *sig, size_t *sig_len,
     return( 0 );
 }
 
+#endif
```

Comment:
Minor: We usually add a comment indicating which guard is being closed: `#endif /* MBEDTLS_ECDSA_C */`.

Reply:
This seems to remain open.